### PR TITLE
Ux login 5 to main

### PR DESCRIPTION
**Pull request by CHOCOLVT**

### CODE REVIEW

**Comment by JorisMonnet** [comment]
The test not passing is being corected in PR #135 when merged, it will works 👍

**Review by JorisMonnet — APPROVED** [review]
You need to add some kdoc 

your testBirthDAteBtn don't assert anything

File: app/src/androidTest/java/ch/epfl/sdp/blindwar/SplashScreenActivityTest.kt [hunk 1]
```diff
@@ -40,16 +32,10 @@ class SplashScreenActivityTest : TestCase() {
...
     @Test
-    fun testOnCreate() {
-        // What The FFF
-//        if (UserAuth().isSignedIn()) {
-//            Intents.intended(IntentMatchers.hasComponent(MainMenuActivityTest::class.java.name))
-//        } else {
-//            Intents.intended(IntentMatchers.hasComponent(NewUserActivityTest::class.java.name))
-//        }
-   }
+    fun testLayoutVisibility() {
```
Comment (ghemDD) [inline app/src/androidTest/java/ch/epfl/sdp/blindwar/SplashScreenActivityTest.kt:36]: Empty test

File: app/src/androidTest/java/ch/epfl/sdp/blindwar/NewUserActivityTest.kt [hunk 2]
```diff
@@ -40,12 +41,54 @@ class NewUserActivityTest : TestCase() {
...
+        for (id in visible_ids) {
+            onView(withId(id))
+                .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
+        }
+    }
+
+    @Test
+    fun testConfirm_allGood() {
+        onView(withId(R.id.NU_pseudo))
+            .perform(replaceText("ValidPseudo"))
```
Comment (marluxiaboss) [inline app/src/androidTest/java/ch/epfl/sdp/blindwar/NewUserActivityTest.kt:61]: It's better to create a variable forr the strings in the test if it's reusable.

**Review by ghemDD — APPROVED** [review]

**Review by marluxiaboss — APPROVED** [review]
You should maybe comment you code more cleanly with /* */ for multi-line comments.

**Pull request merged**